src/backends/plonky2/recursion/circuit.rs

```diff
@@ -702,128 +672,153 @@ mod tests {
     #[test]
     fn test_recursive_circuit() -> Result<()> {
         let arity: usize = 2;
+        let num_public_inputs: usize = 4;
 
         type RC<I> = RecursiveCircuit<I>;
         let inner_params = ();
-        let params: RecursiveParams = new_params::<Circuit1>(arity, &inner_params)?;
+        // let params: RecursiveParams =
+        //     new_params::<Circuit3>(arity, num_public_inputs, &inner_params)?;
```

Comment:
rm? (together with other commented lines from this test)

Reply:
oops! fixed in https://github.com/0xPARC/pod2/pull/243/commits/df2c1efcf30668e98c04b6c45485998f098eaa25